### DISCUSSION TITLE
Can I launch stable-diffusion-v1.5 in xinference, and add the url in open-webui?

### DISCUSSION THREAD

**Discussion by Fruit-GG** [open-webui/open-webui#6884] [General]
# Feature Request
Hello everyone, I have deployed xinference already. I want to launch stable-diffusion-v1.5 on it and add it as the AUTOMATIC1111 base url.
I have set URL：**http://172.16.149.132:25622/sdapi/v1/txt2img** ,the default model is "**stable-diffusion-v1.5**",However it pop out that
```
Something went wrong :/ string indices must be integers, not 'str'
Oops! The URL you provided is invalid. Please double-check and try again.
```

The log is as follow:
```
INFO:     172.16.149.132:52555 - "POST /images/api/v1/config/update HTTP/1.1" 200 OK
INFO:     172.16.149.132:52555 - "GET /api/config HTTP/1.1" 200 OK
INFO  [open_webui.env] Saving 'ENABLE_IMAGE_GENERATION' to the database
INFO  [open_webui.env] Saving 'ENABLE_IMAGE_GENERATION' to the database
INFO:     172.16.149.132:59409 - "GET /images/api/v1/config/url/verify HTTP/1.1" 400 Bad Request
INFO:     172.16.149.132:52555 - "GET /images/api/v1/models HTTP/1.1" 400 Bad Request

```

It doesn't work even I set base URL `http://172.16.149.132:25622 `

What I need to develop to make it available to xinference?

**Comment by NullEqualsZero** [comment]
did you find out what was going on?
